src/pages/CreateSomeTest/testsetup.module.scss

```diff
@@ -0,0 +1,153 @@
+.box{
+    display: flex;
+    justify-content: center;
+    background-color: cadetblue;
+    flex-direction: column;
+    width: 100%;
+    padding:120px 30px
+    
+}
+
+.boxNotRight{
+    display: flex;
+    justify-content: center;
+    background-color: rgb(234, 12, 12);
+    flex-direction: column;
+    width: 100%;
+    padding:120px 30px
+}
+
+.select{
+    max-width: 400px;
+    margin-bottom: 40px;
+}
+$primary: #e7d422;
```

Comment:
if u need some css variable, then create separate file for them and reuse it on all project